lib/travis/build/git/clone.rb

```diff
@@ -101,8 +101,8 @@ def checkout
           end
 
           def checkout_ref
-            return 'FETCH_HEAD' if data.pull_request
-            return data.tag     if data.tag
+            return 'FETCH_HEAD'         if data.pull_request
+            return data.tag.shellescape if data.tag
```

Comment:
@DamianSzymanski I think what @cotsog is saying is that this line can be rewritten to:

```ruby
return tag if data.tag
```